README.md

```diff
@@ -144,8 +144,8 @@ topaz run
 Retrieve the "Citadel" json files, placing them in the current directory:
 
 ```shell
-curl https://raw.githubusercontent.com/aserto-dev/topaz/main/assets/citadel-objects.json >./citadel-objects.json
-curl https://raw.githubusercontent.com/aserto-dev/topaz/main/assets/citadel-relations.json >./citadel-relations.json
+curl https://raw.githubusercontent.com/aserto-dev/topaz/main/assets/citadel-objects-v3.json >./citadel-objects.json
```

Comment:
We should pull from https://github.com/aserto-dev/topaz/tree/main/assets/citadel which contains v3 data files and manifest file.

Reply:
done